metrics/processors/node_autoscaling_enricher.go

```diff
@@ -49,6 +49,19 @@ func (this *NodeAutoscalingEnricher) Process(batch *core.DataBatch) (*core.DataB
 			capacityMem, _ := node.Status.Capacity[kube_api.ResourceMemory]
 			allocatableCpu, _ := node.Status.Allocatable[kube_api.ResourceCPU]
 			allocatableMem, _ := node.Status.Allocatable[kube_api.ResourceMemory]
+			if capacityNvidia, exists := node.Status.Capacity[kube_api.ResourceNvidiaGPU]; exists {
```

Comment:
As I said before, this is deprecated and the resource management WG doesn't want to add more code that depends on this. Please get rid of this metric completely.

Reply:
Will do.